odoo/tools/float_utils.py

```diff
@@ -90,6 +93,17 @@ def float_round(value, precision_digits=None, precision_rounding=None, rounding_
         normalized_value += sign*epsilon
         rounded_value = math.floor(abs(normalized_value)) * sign
 
+    # TIE-BREAKING: HALF-EVEN
+    # We want to apply HALF-EVEN tie-breaking rules, i.e. 0.5 rounds towards closest even number.
+    elif rounding_method == 'HALF-EVEN':
+        rounded_value = math.copysign(builtins.round(normalized_value), normalized_value)
```

Comment:
Why do you need the `copysign`?

Reply:
Because `builtins.round() `will return `0` for both `0` **and** `-0`.